setup

```diff
@@ -275,6 +275,11 @@ if [ ! "\$EXTRA_LDFLAGS" = "" ]; then
   OPTIONS="\$OPTIONS -DHAERO_EXTRA_LDFLAGS=\$EXTRA_LDFLAGS"
 fi
 
+# Add flag to turn off warnings for use of deprecated
+# functions in Kokkos.
+# See eagles-project/mam4xx Issue #99 for more details.
+OPTIONS="\$OPTIONS -DKokkos_ENABLE_DEPRECATION_WARNINGS:BOOL=OFF"
```

Comment:
It might actually be better to set this option explicitly somewhere within [this block](https://github.com/eagles-project/haero/blob/main/ext/CMakeLists.txt#L16), since that's where we set the other Kokkos-related flags.